styles/reset.css

```diff
@@ -85,12 +85,12 @@ time,
 mark,
 audio,
 video {
-    margin: 0;
-    padding: 0;
-    border: 0;
-    font-size: 100%;
-    font: inherit;
-    vertical-align: baseline;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
 }
 
 /* HTML5 display-role reset for older browsers */
```

Comment:
Can we put reset.css into global.css? By the way, where is this reset.css been used? I can not see it in index.tsx

Reply:
This is not part of my card. 
Besides, I think combine reset & global is not a good idea, they are used for different purposes.

Reply:
Hi Harry. I think Cam did not modify reset.css file and it's just some auto-formatting changes from what I committed. We can put this into global.css and it will works fine. When I did this, I just felt the file would be too long to read keeping all default styling reset code and custom styling code in just one file. Besides, we may not modify this reset code again but we might adjust the custom styling code someday, so I decided to keep them in separate files to prevent interfering each other. 
The reset.css is used in _app.tsx that is not in files changed for this commit, so you might not see it.